tests/test_restriction_matrix.cc

```diff
@@ -334,6 +340,6 @@ BOOST_AUTO_TEST_CASE(weight_sum, *utf::tolerance(1e-4))
       e[i] = 1.;
     e.compress(::dealii::VectorOperation::insert);
     restrictor_matrix.vmult(ee, e);
-    BOOST_TEST(ee.l1_norm() == 1.);
+    BOOST_TEST(ee.l1_norm() == 1., tt::tolerance(2e-4));
```

Comment:
So you relaxed the tolerance (x2)?  I missed that change.  Is that how you got the tests to pass?

Reply:
c.c. @Rombur who knows the restriction matrix tests better than I do.  Is that OK?

Reply:
I relaxed the tolerance for restriction test, yes. The tolerances that I was getting from the test when running with Lanczos were around 1.1e-4.

Reply:
Marking it as "resolved".  I just want to make sure that Bruno sees that.  You don't need to wait for his feedback to merge.